CHANGES/7552.feature

```diff
@@ -0,0 +1 @@
+Add `skip_default_headers` to client session requests
```

Comment:
This must use RST, and follow the writing style as explained in https://docs.aiohttp.org/en/stable/contributing.html#adding-change-notes-with-your-prs.